services/src/main/java/org/fao/geonet/api/site/LoggingApi.java

```diff
@@ -224,6 +225,7 @@ private boolean isAppenderLogFileLoaded(FileAppender fileAppender) {
                 return false;
             } else {
                 String logFileName = fileAppender.getFile();
+                // fileAppender found, but no file available ?
```

Comment:
I am not be a big fan of such comments and on the other hand the log message which is following is let vague... Maybe having an explicit log message should exempt from commenting ?